README.md

```diff
@@ -162,18 +162,137 @@ Während der Übertragung der Daten zu den Servern wird in der Statuszeile (unte
- * Zweite Zeile, Sensor Board:
+ * Zweite Zeile, Sensor Board:\
  Hier *esp32* auswählen (über die kleinen Pfeile rechts)
- * Basisinformation:
+ * Basisinformation:\
  Hier die Adresse eingeben (mit dem Land!). Der interne Name des Sensors kann beliebig vergeben werden, muss aber eingegeben werden. Bitte den Haken bei **Indoor-Sensor** setzen, so lange der Sensor wirklich innen ist.
- * Zusätzliche Informationen:
+ * Zusätzliche Informationen:\
  Kann freigelassen werden, darf aber auch ausgefüllt werden.
- * Hardware-Konfiguration:
+ * Hardware-Konfiguration:\
  Hier als Sensor-Typ den Eintrag **Radiation Si22G** (oder ggf. entsprechend) auswählen. Für den zweiten Sensor kann DHT22 stehen bleiben, das ist für uns irrelevant.
- * Position
+ * Position\
  Hier bitte die Koordinaten eingeben, so genau wie möglich (oder über den rechten Knopf die Koordinaten rechnen lassen). Dies wird benötigt, um den Sensor später auf der Karte anzeigen zu können.
 
- Nun mit *Einstellungen speichern* das Ganze beenden. Dann auf der Übersichts-Seite bei diesem Sensor auf *Daten* klicken. Nun steht hinter *Sensor ID* die ID des Sensors. Diese bitte merken: sie wird für die Abfrage bei sensor.community bzw. bei der Anzeige auf https://multigeiger.citysensor.de benötigt (zur Zeit ist das noch laufend in Arbeit - kann/wird also ab und zu ausfallen :wink: ).
+ Nun mit *Einstellungen speichern* das Ganze beenden. Dann auf der Übersichts-Seite bei diesem Sensor auf *Daten* klicken. Nun steht hinter *Sensor ID* die ID des Sensors. Diese bitte merken: sie wird für die Abfrage bei sensor.community bzw. bei der Anzeige auf https://multigeiger.citysensor.de benötigt.
+ 
+## LoRa-Interface
+Um den Multigeiger in das TTN-Netz zu bekommen, sind ein paar vorbereitende Schritte durchzuführen:
 
+* Anlegen des TTN-Devices bei *The Things Network*
+* Übernehmen der Konstanten in den Multigeiger
+* Anmelden bei *sensor.community* (ehemals luftdaten.info)
+* HTTP-Integration
+
+
+### Anlegen eines TTN-Devices
+Das Gerät muss bei TTN (The Things Network) angemeldet werden. Dazu muss zuerst (falls noch nicht vorhanden) ein Account bei TTN amgelegt werden.
+
+#### Account anlegen
+Über <https://account.thethingsnetwork.org/register> auf die Webseite zum Account anlegen gehen. Hier dann einen **USERNAME** vergeben, die **EMAIL ADRESSE** eintragen und auch ein **PASSWORD** vergeben. Dann rechts unten über **Create account** den Account anlegen. Danach kann man sich mit den neuen Daten an der Console anmelden (<https://account.thethingsnetwork.org/users/login>).
+
+#### Applikation anlegen
+Ist man eingelogged, wird über **APPLICATIONS** und **add applications** die neue Applikation angelegt. Folgende Felder müssen ausgefüllt werden:
+
+* **Application ID:**\
+Eine beliebige Bezeichnung für diese Applikation, die darf es aber in dem Netzwerk noch nicht geben (also z.B.: geiger_20200205)
```

Comment:
Hmm, would it be reasonable to put the geiger MAC in here (not a date)?

Reply:
You can put in what you want.

Reply:
yeah, but if you use the date, 2 users doing that at same date will collide.